File: VB/DevExpress.Calendar/Parts/AppointmentEventArgs.vb
Imports Microsoft.VisualBasic
	Imports System
	Imports System.Diagnostics
	Imports DevExpress.Calendar.Model
Namespace DevExpress.Calendar.Parts

	Public Class AppointmentEventArgs
		Inherits EventArgs
		Private _appointment As Appointment
		Public ReadOnly Property Appointment() As Appointment
			Get
				Return _appointment
			End Get
		End Property

		Public Sub New(ByVal appointment As Appointment)
			Debug.Assert(appointment IsNot Nothing)
			_appointment = appointment
		End Sub
	End Class
End Namespace

File: CS/DevExpress.Calendar/Parts/AppointmentEventArgs.cs
namespace DevExpress.Calendar.Parts {
    using System;
    using System.Diagnostics;
    using DevExpress.Calendar.Model;
    
    public class AppointmentEventArgs : EventArgs {
        Appointment _appointment;
        public Appointment Appointment {
            get {
                return _appointment;
            }
        }

        public AppointmentEventArgs(Appointment appointment) {
            Debug.Assert(appointment != null);
            _appointment = appointment;
        }
    }
}
